src/LayoutManager/LayoutManager.ts

```diff
@@ -59,14 +64,14 @@ export class LayoutManager {
   /**
    * subscribe to object layout triggers
    */
-  protected subscribe(object: FabricObject, context: StrictLayoutContext) {
+  protected subscribe(object: FabricObject, context: { target: Group }) {
     const { target } = context;
     this.unsubscribe(object, context);
     const disposers = [
       object.on('modified', (e) =>
         this.performLayout({
           trigger: 'modified',
-          e: { ...e, target: object },
+          e,
```

Comment:
in the object modified, target is already available in the event, we don't need to repeat it. At leat this says typescript

Reply:
on line 88 ( now 93 ) we do the same for the other event, but in my opinion we shouldn't. If we want to forward the original event that is fine, but we are adding the target itself in the object event, we don't need to copy it in the forwarded event.

Reply:
I remember something about this.
If you wish to do this you should add tests to verify this.
Notice that we add a `target` prop to this object.

Reply:
I see that there is a target prop to this object, that is why i don't see any reason to duplicate it on the event.
Probably also e should be called 'triggerEventData' or something that can let us know what 'e' is because is not clear. The only purpose of this 'e' is to be found later inside 'layout:after' event, correct? What else?